.github/dependabot.yml

```diff
@@ -159,3 +159,81 @@ updates:
     target-branch: "release/0.7"
     ignore:
       - dependency-name: "postgres"
+
+  # Manage dependencies on the release/0.8 branch
```

Comment:
I'm not sure we need to do this: we don't intend to deploy `release/0.8` anywhere, which means we don't really care if its dependencies contain vulnerabilities. And the thing is that running dependabot on an extra release branch does consume GitHub Actions runner time (which we pay for) and in some cases adds human toil for wrangling dependabot PRs.

So instead of adding this stanza, we could just not, and mark the branch as unsupported in the top-level README.md.